### PR TITLE
add exceptions to handle overflow bugs 

### PR DESCRIPTION
**Pull request by kangmingtay**

### CODE REVIEW

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -53,6 +54,8 @@
  */
 public class Parser {
 
+    public static final int PRICE_STRING_MAX_LENGTH = 5;
```
Comment (px1099) [inline src/seedu/addressbook/parser/Parser.java:57]: Too short
The price String is in the format $A.BC
For example: "$5.00" has the length of 5

File: src/seedu/addressbook/common/Messages.java [hunk 2]
```diff
@@ -6,6 +6,7 @@
 public class Messages {
 
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+    public static final String MESSAGE_PRICE_LIMIT_EXCEEDED = "Price cannot be so large! ";
```
Comment (px1099) [inline src/seedu/addressbook/common/Messages.java:9]: Incorrect message
First of all, we don't know if the input is even a number (the user can just input "asdfghjklsdfghjkl") or if it is indeed too large or not (the user type "$000000000000000001.00")
Should be changed to "The price input has cannot have more than <PRICE_STRING_MAX_LENGTH> digits/symbols/characters"

File: src/seedu/addressbook/parser/Parser.java (hunk 1 above)
Comment (px1099) [reply src/seedu/addressbook/parser/Parser.java:57]: Recommendation: if the max price cannot exceed $10000.00, then max length should be 8 (length of $9999.99)

**Review by px1099 — CHANGES_REQUESTED** [review]

**Comment by kangmingtay** [comment]
decided on another implementation, please refer to pull request #200

**Pull request closed without merging**